stattlepy/Stattleship_API.py

```diff
@@ -41,21 +46,54 @@ def ss_get_results(self,**kwargs):
                                 page = value 
                         elif str(key) == 'verbose':
                                 verbose = value
+                        elif str(key) == 'stat_type':
+                             param['type'] = value
                         else:
                                 param[key] = value
                 
-                
-                if(verbose):
-                    print 'Making Initial API Request'
+                ### initial verbose to indicate request occurring
+                if verbose:
+                    print'Making Initial API Request'
                     
-                ###NOTE WALK IS NOT YET SUPPORTED
-                        
-                tmp = self.query_api(sport, league, ep, param, version, walk, page, verbose, token )
+                ### initial query       
+                tmp, return_header = self.query_api(sport, league, ep, param, version, walk, page, verbose, token)
                
+                ### make response list
                 response = list()
                
-                response.append(tmp)       
-               
+                ### set the original first parsed 
+                response.append(tmp)
+                
+                ### walk function IN PROGRESS
+                if(walk):
+                    ### set number of pages to walk through
+                    if 'total' in return_header:
+                        total_results = float(return_header['total'])
+                        rpp = float(return_header['per-page'])
+                        pages = int(math.ceil(total_results/rpp))
+                    else:
+                        pages = 1
+                        total_results = 1
+                        
+                    ### append the results from pages 2+ to the response list
+                    if 'link' in return_header:
+                        for p in range(2,pages+1):
```

Comment:
Sure, I can use those header links.  I was basing mine off of the R wrapper because i figured that had been vetted a little bit.  I'll do some work to change it.